### DISCUSSION TITLE
微信扫码后报错，大佬们帮忙看看

### DISCUSSION THREAD

**Discussion by junyi12340** [zhayujie/chatgpt-on-wechat#1376] [Q&A]
/
```
(eog:19758): dconf-WARNING **: 02:12:29.041: failed to commit changes to dconf: 执行子进程“dbus-launch”失败（没有那个文件或目录）

(eog:19758): dconf-WARNING **: 02:12:29.042: failed to commit changes to dconf: 执行子进程“dbus-launch”失败（没有那个文件或目录）
Please press confirm on your phone.
[ERROR][2023-08-24 02:12:44][app.py:52] - App startup failed!
[ERROR][2023-08-24 02:12:44][app.py:53] - 'wxsid'
Traceback (most recent call last):
  File "/chatgpt-on-wechat/app.py", line 50, in run
    channel.startup()
  File "/chatgpt-on-wechat/channel/wechat/wechat_channel.py", line 118, in startup
    itchat.auto_login(
  File "/chatgpt-on-wechat/lib/itchat/components/register.py", line 38, in auto_login
    self.login(enableCmdQR=enableCmdQR, picDir=picDir, qrCallback=qrCallback,
  File "/chatgpt-on-wechat/lib/itchat/components/login.py", line 66, in login
    status = self.check_login()
  File "/chatgpt-on-wechat/lib/itchat/components/login.py", line 159, in check_login
    if process_login_info(self, r.text):
  File "/chatgpt-on-wechat/lib/itchat/components/login.py", line 211, in process_login_info
    core.loginInfo['wxsid'] = core.loginInfo['BaseRequest']['Sid'] = cookies["wxsid"]
KeyError: 'wxsid'
```

**Comment by zhayujie** [comment]
账号实名了嘛

**Reply by junyi12340** [reply]
大佬实名好了，但是因为我是部在了虚拟机上，网络好像有点问题
报错信息：
Error communicating with OpenAI: Proxy URL had no scheme, should start with http:// or https://
[INFO][2023-08-24 21:00:50][wechat_channel.py:189] - [WX] sendMsg=Reply(type=ERROR, content=[ERROR]
我连接不到你的网络), receiver=@@24b1d98165751d1035aa4b4fc129b968a808ab2d7a4fa99f865d203ba1b3b52b

**Reply by zhayujie** [reply]
链接貌似不对， 你的 proxy 或者 open_ai_api_base 配置有改过嘛

**Reply by Tyuts** [reply]
core.loginInfo['wxsid'] = core.loginInfo['BaseRequest']['Sid'] = cookies["wxsid"]
KeyError: 'wxsid'
记事本里没有找到填入AIP BASE的地方

**Comment by dorothy-in-ds** [comment]
请问解决了吗？遇到了同样的问题

**Comment by Tyuts** [comment]
没有解决




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Huiwen(Dorothy) ***@***.***&gt;; 
发送时间: 2024年6月18日(星期二) 凌晨2:29
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [zhayujie/chatgpt-on-wechat] 微信扫码后报错，大佬们帮忙看看 (Discussion #1376)















请问解决了吗？遇到了同样的问题

-
直接回复这封邮件，在GitHub上查看，或取消订阅.
***@***.***和&gt;
